src/utils/dateUtils.js

```diff
@@ -36,15 +44,3 @@ export const convertPickedDateToUtc = (date) => {
   newDate.setHours(newDate.getHours() - timezoneOffset / 60 - 12);
```

Comment:
Does this work properly when the current time is earlier than the timezoneOffset? (e.g., if timezoneOffset is 8 hours, does this do what you want when the time is 7am?)

In general, it's best to do arithmetic on date or time objects (add/subtract days/hours/minutes/whatever) and then convert to printable.

Reply:
To be more precise:
```
d = new Date();  // gets current time
e = new Date(d.getTime() + 1);  // gets current time + 1 second
```
Doing arithmetic this way deals with wrap-around problems - e.g. `d.setHours(-1)` doesn't do what you expect.

Reply:
what do you mean? I am doing it that way... date.setHours(date.getHours - x)

haven't thoroughly tested this yet but seems to work; i need to update/create tests. i was getting overwhelmed with trying to figure out tests, so i was just focusing on getting the dang thing to work first.